region/lowernorfair/east.json

```
@@ -9521,7 +9532,8 @@
                         }},
                         {"heatFrames": 620}
                       ]}
-                    ]}
+                    ]},
+                    {"shinespark": {"frames": 35}}
```

I don't think this belongs here.
```suggestion
                    ]}
```